File: 2.0/python/contacts/updateContact.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
client.contacts.updateContact(
    id=11646003,
    body={
        'mobilePhone': '12136600123',
        'workPhone': '14553320089',
    }
).result()

File: 2.0/php/contacts/updateContact.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->updateContact();
        $request->getOperationConfig()->setHeaderParameters(array("Content-Type" => "application/json"));
        $request->getOperationConfig()->setPathParameters(array("id" => 11646003));
        $body = '{
                    "mobilePhone":"12136600123",
                    "workPhone":"14553320089"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
